squire_web/src/tournament/players/selected.rs

```diff
@@ -19,16 +25,85 @@ impl SelectedPlayer {
         digest
     }
 
+    pub fn update_round(&mut self, rid: Option<RoundId>) -> bool {
+        let digest = self.round_id != rid;
+        self.round_id = rid;
+        digest
+    }
+
+    fn subview_match(&self, tourn: &Tournament) -> Html {
```

Comment:
It seems like a lot of this is duplicated from the `SelectedRound` in `src/tournament/rounds/selected.rs`. We should figure out an abstraction that will reduce this duplication